tko_web_sessions_management/ir_sessions.py

```diff
@@ -98,16 +98,16 @@ def _on_session_logout(self, logout_type=None):
         cr.autocommit(True)
 
         for session in self:
-            session_duration = now - datetime.strptime(
+            session_duration = str(now - datetime.strptime(
                             session.date_login,
-                            DEFAULT_SERVER_DATETIME_FORMAT)
+                            DEFAULT_SERVER_DATETIME_FORMAT))
             session.sudo().write(
                 {
                     'logged_in': False,
                     'date_logout': now,
                     'logout_type': logout_type,
                     'user_kill_id': SUPERUSER_ID,
-                    'session_duration': session_duration,
+                    'session_duration': session_duration.split('.')[0],
```

Comment:
We need this change as session_duration is a char field by nature and in v10 we are getting duration value eg. 0:17:12.45678

If i don't split it then as the field size is 8 in py definition by default it takes "0:17:12.". So just to remove the trailing (.) I applied this logic here.